doc/releases/changelog-dev.md

```diff
@@ -62,410 +16,352 @@
   [(#2663)](https://github.com/PennyLaneAI/pennylane/pull/2663)
   [(#2684)](https://github.com/PennyLaneAI/pennylane/pull/2684)
   [(#2688)](https://github.com/PennyLaneAI/pennylane/pull/2688)
+  [(#2695)](https://github.com/PennyLaneAI/pennylane/pull/2695)
+  [(#2710)](https://github.com/PennyLaneAI/pennylane/pull/2710)
+  [(#2712)](https://github.com/PennyLaneAI/pennylane/pull/2712)
+
+
+  This includes two new QNode measurements,
+
+  - The Von Neumann entropy via `qml.vn_entropy`:
+  
+    ```pycon
+    >>> dev = qml.device("default.qubit", wires=2)
+    >>> @qml.qnode(dev)
+    ... def circuit_entropy(x):
+    ...     qml.IsingXX(x, wires=[0,1])
+    ...     return qml.vn_entropy(wires=[0], log_base=2)
+    >>> circuit_entropy(np.pi/2)
+    1.0
+    ```
+
+  - Mutual information via `qml.mutual_info`:
+  
+    ```pycon
+    >>> dev = qml.device("default.qubit", wires=2)
+    >>> @qml.qnode(dev)
+    ... def circuit(x):
+    ...     qml.IsingXX(x, wires=[0,1])
+    ...     return qml.mutual_info(wires=[0], wires1=[1])
+    >>> circuit(np.pi/2)
+    tensor(1.38629436, requires_grad=True)
+    ```
+
+  As well as new transforms available in the `qml.qinfo` module,
+
+  - Classical and quantum fisher information transforms:
+  
+    ```python3
+    dev = qml.device("default.qubit", wires=3)
+
+    @qml.qnode(dev)
+    def circ(params):
+        qml.RY(params[0], wires=1)
+        qml.CNOT(wires=(1,0))
+        qml.RY(params[1], wires=1)
+        qml.RZ(params[2], wires=1)
+        return qml.expval(qml.PauliX(0) @ qml.PauliX(1) - 0.5 * qml.PauliZ(1))
+
+    params = np.array([0.5, 1., 0.2], requires_grad=True)
+    cfim = qml.qinfo.classical_fisher(circ)(params)
+    qfim = qml.qinfo.quantum_fisher(circ)(params)
+    ```
+
+    These quantities are typically employed in variational optimization schemes to tilt the gradient in a more favourable direction (see [2103.15191](https://arxiv.org/abs/2103.15191) and [1909.02108](https://arxiv.org/abs/1909.02108)) --- producing what is known as the *natural* gradient. For example:
+
+    ```pycon
+    >>> grad = qml.grad(circ)(params)
```

Comment:
It used to be here, although these days I am very cognizant of ensuring our examples are as concise as possible 😆 Since the main purpose here is to show how the CFIM may be used to construct the natural gradient, I think comparing to the actual gradient is not necessary.